File: backend/src/main/java/com/techgarage/service/ReviewService.java
package com.techgarage.service;

import com.techgarage.dto.review.ReviewRequest;
import com.techgarage.dto.review.ReviewResponse;
import java.util.List;

public interface ReviewService {
    ReviewResponse submit(Long jobId, ReviewRequest request);
    List<ReviewResponse> getForFreelancer(Long freelancerId);
}

File: TechGarage-clean/frontend/src/services/reviewService.js
import api, { unwrap } from './api'

export const reviewService = {
  submit: (jobId, payload) => unwrap(api.post(`/jobs/${jobId}/review`, payload)),
  getForFreelancer: (freelancerId) => unwrap(api.get(`/freelancers/${freelancerId}/reviews`)),
}
